config/config.go

```diff
@@ -218,11 +219,17 @@ type HDFSArtifactRepository struct {
 	Force bool `json:"force,omitempty"`
 }
 
-// PrometheusConfig defines a config for a metrics server
-type PrometheusConfig struct {
-	Enabled       bool   `json:"enabled,omitempty"`
-	DisableLegacy bool   `json:"disableLegacy"`
-	MetricsTTL    TTL    `json:"metricsTTL"`
-	Path          string `json:"path,omitempty"`
-	Port          string `json:"port,omitempty"`
+// MetricsConfig defines a config for a metrics server
+type MetricsConfig struct {
+	// Enabled controls metric emission. Default is true, set "enabled: false" to turn off
+	Enabled *bool `json:"enabled,omitempty"`
```

Comment:
Metrics are now enabled by default. I don't think they would have a large memory footprint if no custom metrics are used.

Open to opinions though.